### ISSUE TITLE
Failed to create process

### ISSUE THREAD

**Issue by DBuckley101** [Unity-Technologies/ml-agents#1845]
I was able to call the `mlagents-learn` command and it popped up with the info saying to press the play button. I did, but then this error occurred.

![image](https://user-images.githubusercontent.com/44452488/54650402-f89f1c80-4af9-11e9-9be8-bfe3401056cd.png)

I stopped playing it in the editor, but going back to the terminal and trying to call the learn command again - just returned this error.

![image](https://user-images.githubusercontent.com/44452488/54650482-7400ce00-4afa-11e9-9768-3d2f0772a591.png)

I'm unable to call any of the mlagents-learn commands as they all return _failed to create process._

**Comment by crawelsky** [comment]
Hi @DBuckley101 , I got the same issue. I solved it by remove HTTPS_proxy and HTTP_proxy in environment variables. Hope that will help you !

**Comment by awjuliani** [comment]
Hi @DBuckley101 

Please let us know if @crawelsky 's solution works for you!

**Comment by DBuckley101** [comment]
I don't have HTTPS_proxy or HTTP_proxy in my environment variables. Now when I try to call the 'mlagents-learn' command, it says it's not a recognized command. 

![image](https://user-images.githubusercontent.com/44452488/54724451-f9e24f00-4bb6-11e9-9050-ad9885afe3b4.png)

**Comment by awjuliani** [comment]
Hi @DBuckley101 

It looks like you ran `mlagents-train` rather than `mlagents-learn`.

**Comment by DBuckley101** [comment]
Ha yeah, still 'failed to create process' unfortunately.

**Comment by eshvk** [comment]
@awjuliani

**Comment by xiaomaogy** [comment]
@awjuliani

**Comment by xiaomaogy** [comment]
Thanks for reaching out to us. Hopefully you were able to resolve your issue.  We are closing this due to inactivity, but if you need additional assistance, feel free to reopen the issue.

**Comment by ZeGaly** [comment]
Hi, 

I would like to re open the subject, sorry if it's a noob issue but I'm struggling for few days now on exactly the same issue of cawelsky. The solution of DBuckley101 doesn't work for me as there is no proxy variable in my environment variables. 

I installed and run successfully the training for 3Dball example 1 month ago. I developed my own little game but I faced this issue. I got back at 3Dball to see if it was my own game the problem but I have still the issue with 3Dball now. 
I think the issue is  global as I have the same failed message when I try to run the --help command. And the .conda environment is created and the mlagents--learn command is in some way recognized as when I test another kind of command, the command is not recognized (I have well activated the ml-agents environment). 

Any idea ? 
Thanks a lot for your help
